.github/workflows/publish.yml

```diff
@@ -0,0 +1,22 @@
+name: publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  publish:
+    if: github.ref == 'refs/heads/main'
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
```

Comment:
Can you use Node 16 here?

Reply:
https://github.com/DopplerUniversity/gitops-secrets-nodejs/pull/14